Sources/BSON/ExtendedJSONDecoder.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+/// Decoder for ExtendedJSON
+public class ExtendedJSONDecoder {
+    /// Initialize a Decoder
```

Comment:
complete sentence needed here as well, and let's use the full type name?

Reply:
done